### PR TITLE
Rename latest to insiders

### LINKED ISSUE #5830: [VS Code] Signal to users that your are using the insiders build (or latest)
I had switched to latest some time back and now got surprised by a non-functional IDE.
Users should be reminded somehow that they are running latest.

### PR DESCRIPTION
**Pull request by jeanp413**
## Related Issue(s)

Fixes #5830

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **jeanp413** and additionally assign **geropl** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @geropl` in a comment when ready.

Associated issue: *#5830*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**
- **[components/gitpod-protocol/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-protocol/OWNERS)**
- **[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **jeanp413** and additionally assign **gtsiolis** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @gtsiolis` in a comment when ready.

Associated issue: *#5830*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**
- **[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by filiptronicek — COMMENTED** [review]
Loving this change, makes it a lot clearer that this is not stable, however, the PR title mentions **Insiders**, but the code mentions **Insider**, is that a typo or is it intentional to make the title shorter?

File: components/dashboard/src/settings/Preferences.tsx
Comment (bigint) [inline components/dashboard/src/settings/Preferences.tsx:58]:
```suggestion
                        <span className="mt-2 ml-2 self-center rounded-xl py-0.5 px-2 text-sm bg-orange-100 text-orange-700 dark:bg-orange-600 dark:text-orange-100 font-semibold">INSIDERS</span>
```

I think it is intended to be `INSIDERS` - https://code.visualstudio.com/insiders

image

Comment (bigint) [reply components/dashboard/src/settings/Preferences.tsx:58]: Oooh! @filiptronicek already suggested it ❤️

**Review by filiptronicek — APPROVED** [review]

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: db46a554efbbe8e5b3770cd57a4ecd7838b5042d</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5972#pullrequestreview-769019670" title="Approved">filiptronicek</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **jeanp413** and additionally assign **gtsiolis** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @gtsiolis` in a comment when ready.

Associated issue: *#5830*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**
- **[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/ide/code/leeway.Dockerfile
Comment (jankeromnes) [inline components/ide/code/leeway.Dockerfile:32]: FYI, this commit hasn't been merged to branch `gp-code` yet: https://github.com/gitpod-io/openvscode-server/pull/126 -- should we wait for a merge?

**Comment by jankeromnes** [comment]
Many thanks @jeanp413 for this fix and @filiptronicek + @bigint for the reviews! 🙏

Also looks good to me, approving on behalf of the meta team:

/approve

Adding a hold to pause auto-merge, in case we want to wait for https://github.com/gitpod-io/openvscode-server/pull/126 to be merged first (please feel free to use `/hold cancel` when you'd like this to be merged)

/hold

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5972#pullrequestreview-769019670" title="Approved">filiptronicek</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5972#issuecomment-932281520" title="Approved">jankeromnes</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **jeanp413** after the PR has been reviewed.

Associated issue: *#5830*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [jankeromnes]
- **[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by jankeromnes** [comment]
> This PR is **NOT APPROVED**

> please ask for approval from **jeanp413** after the PR has been reviewed.

Ah, I guess you must also `/approve` your own PR @jeanp413 🙄 seems kind of silly.

File: components/ide/code/leeway.Dockerfile
Comment (jeanp413) [reply components/ide/code/leeway.Dockerfile:32]: yeah I have some thoughts on the changes in openvscode-server side that I want to double check with Anton first when he's back on Monday

**Comment by akosyakov** [comment]
@gtsiolis @svenefftinge Do you think it would be enough of the difference to understand, i.e. you can only find it in About dialog: https://github.com/gitpod-io/openvscode-server/pull/126#pullrequestreview-769041446 It is the same in VS Code Desktop for Insiders, but there is also another icon for the app.

**Comment by akosyakov** [comment]
@jeanp413 I think we can go on and merge these PRs for now.

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5972#pullrequestreview-769019670" title="Approved">filiptronicek</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5972#issuecomment-932281520" title="Approved">jankeromnes</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5972#issuecomment-933275443" title="Approved">JanKoehnlein</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **jeanp413** after the PR has been reviewed.

Associated issue: *#5830*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [JanKoehnlein,jankeromnes]
- **[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
New changes are detected. LGTM label has been removed.

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5972#pullrequestreview-769019670" title="Approved">filiptronicek</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5972#issuecomment-932281520" title="Approved">jankeromnes</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5972#issuecomment-933275443" title="Approved">JanKoehnlein</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5972#issuecomment-933706140" title="Approved">jeanp413</a>*

Associated issue: *#5830*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [JanKoehnlein,jankeromnes]
- ~~[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)~~ [jeanp413]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by gtsiolis** [comment]
> ...  but there is also another icon for the app.

❓ **question**: @akosyakov how can I see this new icon? Do we want to use the VS Code Insiders icon in `/preferences`?

**Comment by gtsiolis** [comment]
**fyi**: @jeanp413 this still needs the `lgtm` label to for automatic merge. 💡

**Comment by jeanp413** [comment]
yeah, waiting for someone @akosyakov @filiptronicek @jankeromnes to add `lgtm` again.

**Review by filiptronicek — APPROVED** [review]
LGTM

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 2950871b99c9a66fcf3064c44ce256319b619fff</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5972#pullrequestreview-770995536" title="Approved">filiptronicek</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5972#issuecomment-932281520" title="Approved">jankeromnes</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5972#issuecomment-933275443" title="Approved">JanKoehnlein</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5972#issuecomment-933706140" title="Approved">jeanp413</a>*

Associated issue: *#5830*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [JanKoehnlein,jankeromnes]
- ~~[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)~~ [jeanp413]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
